diff --git a/debian.qcom-x1e/config/annotations b/debian.qcom-x1e/config/annotations
index 195233568..d50f835ba 100644
--- a/debian.qcom-x1e/config/annotations
+++ b/debian.qcom-x1e/config/annotations
@@ -12879,7 +12879,7 @@ CONFIG_VEML6040                                 policy<{'arm64': 'm'}>
 CONFIG_VEML6046X00                              policy<{'arm64': 'm'}>
 CONFIG_VEML6070                                 policy<{'arm64': 'm'}>
 CONFIG_VEML6075                                 policy<{'arm64': 'm'}>
-CONFIG_VERSION_SIGNATURE                        policy<{'arm64': '""'}>
+CONFIG_VERSION_SIGNATURE                        policy<{'arm64': '"Ubuntu 7.2-rc5-jg-0-qcom-x1e 7.2-rc5"'}>
 CONFIG_VETH                                     policy<{'arm64': 'm'}>
 CONFIG_VEXPRESS_CONFIG                          policy<{'arm64': 'y'}>
 CONFIG_VF610_ADC                                policy<{'arm64': 'm'}>
